Tests/HTTPTests/ServerTests.swift

```diff
@@ -533,6 +535,7 @@ class ServerTests: XCTestCase {
         ("testSimpleHelloEndToEnd", testSimpleHelloEndToEnd),
         ("testRequestEchoEndToEnd", testRequestEchoEndToEnd),
         ("testRequestKeepAliveEchoEndToEnd", testRequestKeepAliveEchoEndToEnd),
+        ("testMultipleSimultaneousRequestsEchoEndToEnd",testMultipleSimultaneousRequestsEchoEndToEnd),
```

Comment:
Minor nit - should have a space.

Reply:
There's also whitespace issues on lines 285->287 which could be fixed up at the same time.